docker/images/elastic_news_data/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM elasticsearch:2.1.1
+
+COPY data /usr/share/elasticsearch/data
+
+RUN chown -R elasticsearch:elasticsearch /usr/share/elasticsearch/data
+
+VOLUME /usr/share/elasticsearch/data
+
+CMD tail -f /dev/null
```

Comment:
This keeps the container running forever, as it would otherwise exit immediately as it has no real program. This isn't *really* needed in docker-compose, but it makes it easier to understand as it stays running forever alongside the containers with programs in them, and is actually stopped along with the other cotntainers.